src/containers/PartyView.js

```diff
@@ -103,15 +103,23 @@ function PartyView({ location }) {
         { characters.length > 1 && (
           <>
             <SideBar className={sidebar ? 'open' : ''}>
-              <Card style={{ margin: '1rem' }}>
+              <Card style={{ margin: '.25rem', marginTop: '2rem' }}>
                 <PartyActions>
                 {existingParty ? <h2>{partyName}</h2>
-                  : <form onSubmit={saveParty}>
+                  : <form>
                           <TextField onChange={(event) => { setPartyName(event.target.value); }} label="PartyName" />
-                          <Button color="primary" variant="contained" type="submit">Save Party</Button>
+                          {partyName.length > 0
+                          && <i
+                            className="material-icons"
+                            onClick={saveParty}
+                            style={{
+                              float: 'right', marginTop: '1rem', color: 'darkslateblue', cursor: 'pointer',
+                            }}
+                          >save
+                             </i> }
```

Comment:
2 things, 

1. I really want to start cutting out inline styling. It gets hard to maintain fast.
2. Most of this styling should be taken care of with existing Components
The IconButton Component should take care of color and cursor, and the postisioning should just be handled in Flex, thats what the Spacer component is for in styled components. If you put it between two items in a flex row it will expand to take up all the middle space.

What do you think?

Reply:
1. Yes, what's the protocol with styled components? Just have classes like usual? For the weird exceptions anyway?

2. Sounds good

Reply:
For styled components, if they're a one-off they just go at the bottom of that file for now. If its reusable it goes in the [CustomStyled.js](https://github.com/kjintroverted/dragon-ui/blob/master/src/components/CustomStyled.js) and imported

Reply:
okay so how do you do a stylized component that is a material-core component?